tests/testthat/test-post.R

```diff
@@ -1,12 +1,42 @@
 test_that("upload_media_to_twitter() can handle small file", {
   skip_on_cran()
-  
-  id <- upload_media_to_twitter(test_path("tweet.gif"))
+
+  # gif
+  path <- test_path("tweet.gif")
```

Comment:
I pulling `path` out into a separate variable makes the test harder to understand because it introduces indirection.